Third_Edition_Explainer.md

```diff
@@ -105,15 +105,20 @@ was a feature.
 
 Firefox 3 added APNG support in 2007.
 
-The PNG development group, after much discussion, **rejected APNG in 2007**,
-mostly on the grounds that it was not MNG and that PNG should remain a static format.
-Two attempts to restart the discussion in the following years failed.
+The PNG development group, after much discussion, rejected the APNG proposal in 2007
+on the grounds that PNG should remain a static format and that APNG, unlike MNG, did
+not clearly identify itself as a separate format.
 
-APNG was documented [on the Mozilla wiki](https://wiki.mozilla.org/APNG_Specification)
-and browsers and other tools added a variety of patches to their own versions of libpng,
+Two attempts to revisit the decision in the following years failed.
+
+APNG is documented on the [Mozilla wiki](https://wiki.mozilla.org/APNG_Specification)
+and some browsers and other tools added a variety of patches to their own versions of libpng
 to support APNG.
 
-APNG started to be [**added to other platforms**](https://caniuse.com/apng):
+The adoption of APNG is documented by [Alexis Deveria](http://a.deveria.com/) on the "can I use"
+web page [Animated PNG (APNG)](https://caniuse.com/apng).
```

Comment:
CanIUse is very well known in the W3C community and does not really merit the detailed introduction. Not that I am opposed, but it really isn't needed.